.github/workflows/macOS.yml

```diff
@@ -1,5 +1,5 @@
-name: macOS Build
-on: [push, pull_request]
+name: macOS-build
+on: [pull_request push]
```

Comment:
Missing comma. Why is the order changed?

Reply:
Experimentation. I will change it back, though it seems to work fine. ;-)